recipe/yum_requirements.txt

```diff
@@ -1,3 +1,4 @@
 libXt-devel
 mesa-libGLU-devel
 xorg-x11-server-Xvfb
+psmisc
```

Comment:
FYI I needed to add this to get `killall` support. We could in theory use `ps` and pass the result to `kill` or something, but continuing to use `killall` seemed cleaner